docs/sources/flow/reference/components/prometheus.exporter.unix.md

```diff
@@ -135,6 +135,8 @@ The following blocks are supported inside the definition of
 
 ### filesystem block
 
+The default values can vary by the operating system the agent runs on - refer to the [integration source](https://github.com/grafana/agent/blob/main/internal/static/integrations/node_exporter/config.go) for up-to-date values on each OS.
```

Comment:
Should we call out the OS-specific defaults (instead of linking to the source code)?

Reply:
I'd also prefer not linking to the source code here; if you don't want to do that in this PR the work can be split up.

Reply:
I've created https://github.com/grafana/agent/issues/6701 and will follow up on this.